README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+//In a responsive design, one page with an optimal viewing experince on any device, and in Adaptie its multiple layouts for screen sizes
+
```

Comment:
Please elaborate on this - look up the definitions used in the training kit and on Google and reanswer this question